icecap/sysroot/workspace/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "sysroot"
+version = "0.0.0"
```

Comment:
Can you set the `version` number to `0.3.0`, set the `author` field to "The Veracruz development team", and add a short description in the `description` field?

Reply:
Done.